### DISCUSSION TITLE
My Toonsquid keeps crashing whenever I try to open my project..

### DISCUSSION THREAD

**Discussion by Flurisun** [keiwando/toonsquid#611] [Q&A]
I was working on my animation and as soon as I enabled Onion Skin it crashed and I havent been able to open it since. Is there anything anyone can do about this? I am unable to disable Onion Skin :(

**Comment by budaloops** [comment]
Not sure what’s the best solution for this. 

However, you could make a copy the Projects folder (Files —> On My iPad —> ToonSquid —> Projects) then delete ToonSquid and re-install it. Then replace the newly created Projects folder with your copy. 

I have not tried this with ToonSquid but I’ve done similar things with Procreate and Affinity Designer.

If it doesn’t work, well you get to re-do your project and practice more, and that never hurts in the end (because it hurts at the start).

**Comment by keiwando** [comment]
I'm sorry to hear that you are experiencing this!
Please follow these steps for restoring the latest backup and for sending me a copy of the broken project file so that I can investigate what might be wrong with it:
https://toonsquid.com/handbook/projects/backup/#project-corrupted-recovery-steps
